content/blog/2018/data-plane-setup/index.md

```diff
@@ -0,0 +1,533 @@
+---
+title: Demystifying Istio's Sidecar Injection Model 
+description: De-mystify how Istio manages to plugin its data-plane components into an existing deployment. Deep-dive into sidecar injection, how it is done whether manually or `namespaceSelector` based automatic injection. And also how the iptables are modified in the pod namespace to redirect the traffic to the injected proxy. 
+publishdate: 2018-12-12
+subtitle:
+attribution: Manish CHUGTU
+twitter: chugtum 
+weight: 78
+keywords: [kubernetes, istio, sidecar-injection, admission-controller, mutatingwebhook]
+
+---
+A simple overview of an Istio service-mesh architecture always starts with describing the control-plane and data-plane.
+
+_From Istio’s Documentation:_
+
+> An Istio service mesh is logically split into a data plane and a control plane.
+>
+>  The data plane is composed of a set of intelligent proxies (Envoy) deployed as sidecars. These proxies mediate and control all network communication between microservices along with Mixer, a general-purpose policy and telemetry hub.
+>
+>  The control plane manages and configures the proxies to route traffic. Additionally, the control plane configures Mixers to enforce policies and collect telemetry.
+
+{{< image width="40%"
+    ratio="33%"
+    link="./arch-2.svg"
+    alt="The overall architecture of an Istio-based application."
+    caption="Istio Architecture"
+    >}}
+It is important to understand that the sidecar injection into application pods happens automatically, though manual injection is also possible. Traffic is directed from the application services to and from these sidecars without developers needing to worry about it. Once they are connected to the Istio service mesh, they can start using and reaping the benefits of all that it has to offer. But how does the data plane plumbing happen and what is really required to make it work seamlessly? In this post we will deep-dive into the specifics of sidecar injection models for a very clear understanding of how it works.
+
+## Sidecar Injection
```

Comment:
Here and elsewhere, only capitalize the first letter of headers (unless its a proper noun or acronym)

Reply:
Done at all the places.